Samples/office-keyboard-shortcuts/README.md

```diff
@@ -0,0 +1,166 @@
+
+### Run the sample from localhost
+
+If you prefer to host the web server for the sample on your computer, follow these steps.
+
+1. Install a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/) on your computer. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+1. You need http-server to run the local web server. If you haven't installed this yet you can do this with the following command.
+
+    ```console
+    npm install --global http-server
+    ```
+
+1. You need Office-Addin-dev-certs to generate self-signed certificates to run the local web server. If you haven't installed this yet, you can do this with the following command.
+
+    ```console
+    npm install --global office-addin-dev-certs
+    ```
+
+1. Clone or download this sample to a folder on your computer, then go to that folder in a console or terminal window.
+1. Run the following command to generate a self-signed certificate to use for the web server.
+
+   ```console
+    npx office-addin-dev-certs install
+    ```
+
+    This command will display the folder location where it generated the certificate files.
+
+1. Go to the folder location where the certificate files were generated, then copy the **localhost.crt** and **localhost.key** files to the cloned or downloaded sample folder.
+1. Run the following command.
+
+    ```console
+    http-server -S -C localhost.crt -K localhost.key --cors . -p 3000
+    ```
+
+    The http-server will run and host the current folder's files on localhost:3000.
+
+1. Sideload the **manifest-localhost.xml** file in Excel or Word. The sideloading process varies depending on your platform.
+
+    - **Office on the web**: [Manually sideload an add-in to Office on the web](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing#manually-sideload-an-add-in-to-office-on-the-web)
+    - **Office on Windows**: [Sideload Office Add-ins for testing from a network share](https://learn.microsoft.com/office/dev/add-ins/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins)
+    - **Office on Mac**: [Sideload Office Add-ins on Mac for testing](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-an-office-add-in-on-mac)
+1. Verify that the add-in loaded successfully. You'll see a **Keyboard shortcuts** button on the **Home** tab of the ribbon.
+1. Test the sample by following the steps in [Try it out](#try-it-out).
+
+## Try it out
+
+Once the add-in is loaded, try out its functionality.
+
+1. Press **Ctrl+Alt+1** on your keyboard to open the add-in's task pane.
+
+    > [!NOTE]
+    > If the keyboard shortcut is already in use in Excel or Word, a dialog will be shown, so that you can select which action you'd like to map to the shortcut. Once you select an action, you can change your preference by invoking the **Reset Office Add-in Shortcut Preferences** command from the search field.
```

Comment:
```suggestion
    > If the keyboard shortcut is already in use in Excel or Word, a dialog will be shown so that you can select which action you'd like to map to the shortcut. Once you select an action, you can change your preference by invoking the **Reset Office Add-in Shortcut Preferences** command from the search field.
```